boards/mcb2388/Makefile.features

```diff
@@ -0,0 +1,9 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_gpio periph_gpio_irq
```

Comment:
gpio features could be provided at CPU level

Reply:
:+1: This also not done for the other old ARM7 boards. See https://github.com/RIOT-OS/RIOT/pull/12672